tox.ini

```diff
@@ -1,32 +1,45 @@
 [tox]
 envlist =
        py38-{flake8,codecov,docs},
-       {py36,py37,py38}-django{21,22}-drf{39,310,311}
-       {py36,py37,py38}-django{30}-drf{310,311}
+       {py36,py37,py38}-django21-drf{39,310,311}
+       {py36,py37,py38,py39}-django22-drf{39,310,311,312}
+       {py36,py37,py38,py39}-django30-drf{310,311,312}
+       {py36,py37,py38,py39}-django31-drf{311,312}
+       {py36,py37,py38,py39,py310}-django32-drf{312}
+
+[gh-actions]
+python =
+    3.6: py36
+    3.7: py37
+    3.8: py38,flake8,docs
+    3.9: py39
+    3.10: py310
 
 [testenv]
-passenv = CI TRAVIS TRAVIS_*
 setenv =
        PYTHONDONTWRITEBYTECODE=1
 commands = ./runtests.py --fast {posargs} --coverage
 deps =
-       django21: Django==2.1.15
-       django22: Django==2.2.8
-       django30: Django==3.0.0
+       django21: Django~=2.1.15
+       django22: Django~=2.2.8
+       django30: Django~=3.0.0
+       django31: Django~=3.1.0
+       django32: Django~=3.2.0
        drf39: djangorestframework==3.9.4
        drf310: djangorestframework==3.10.3
        drf311: djangorestframework==3.11.0
-       django-test-plus==1.4.0
-       pytest==4.4.1
-       pytest-django==3.4.8
-       pytest-cov==2.6.1
-       hashids==1.1.0
-       mock==2.0.0
+       drf312: djangorestframework~=3.12.0
+       django-test-plus==2.2.0
+       pytest==6.2.5
+       pytest-django==4.4.0
+       pytest-cov==3.0.0
+       hashids==1.3.1
+       mock==4.0.3
```

Comment:
I had to update `pytest` for Python 3.10 support. I took this opportunity to update all pinned versions here.